airbyte-integrations/connectors/source-captain-data/source_captain_data/manifest.yaml

```diff
@@ -0,0 +1,170 @@
+          - path: ["project_uid"]
+            value: "{{ config['project_uid'] }}"
+
+  workflows_stream:
+    $ref: "#/definitions/base_stream"
+    $parameters:
+      name: "workflows"
+      primary_key: "uid"
+      path: "/workflows"
+
+  # Sliced streams
+  jobs_stream:
+    $ref: "#/definitions/base_stream"
+    $parameters:
+      name: "jobs"
+      primary_key: "uid"
+      path: "/workflows/{{ stream_slice.parent_id }}/jobs"
+    retriever:
+      requester:
+        $ref: "#/definitions/requester"
+      partition_router:
+        type: SubstreamPartitionRouter
+        parent_stream_configs:
+          - stream: "#/definitions/workflows_stream"
+            parent_key: "uid"
+            partition_field: "parent_id"
+      record_selector:
+        $ref: "#/definitions/selector"
+
+  successful_jobs_stream:
+    $ref: "#/definitions/base_stream"
+    $parameters:
+      name: "jobs"
+      primary_key: "uid"
+      path: "/workflows/{{ stream_slice.parent_id }}/jobs"
+    retriever:
+      requester:
+        $ref: "#/definitions/requester"
+      partition_router:
+        type: SubstreamPartitionRouter
+        parent_stream_configs:
+          - stream: "#/definitions/workflows_stream"
+            parent_key: "uid"
+            partition_field: "parent_id"
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: []
+        record_filter:
+          condition: "{{ record['total_row_count'] > 0 }}"
+
+  job_results_stream:
+    $ref: "#/definitions/base_stream"
+    $parameters:
+      name: "job_results"
+      primary_key: "job_result_id"
+      path: "/jobs/{{ stream_slice.parent_id }}/results"
+      stream_cursor_field: "extracted_at"
+    retriever:
+      requester:
+        $ref: "#/definitions/requester"
+      paginator:
+        $ref: "#/definitions/captain_data_paginator"
+      partition_router:
+        type: SubstreamPartitionRouter
+        parent_stream_configs:
+          - stream: "#/definitions/successful_jobs_stream"
+            parent_key: "uid"
+            partition_field: "parent_id"
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: ["results"]
+    transformations:
+      - type: AddFields
+        fields:
+          - path: ["results"]
+            value: "{{ record }}"
+          - path: ["job_result_id"]
+            value: "{{ record|hash('md5') }}"
+          - path: ["job_uid"]
+            value: "{{ stream_slice.parent_id }}"
+          - path: ["extracted_at"]
+            value: "{{ now_utc() }}"
+
+streams:
+  - "#/definitions/workspace_stream"
+  - "#/definitions/workflows_stream"
+  - "#/definitions/jobs_stream"
+  - "#/definitions/job_results_stream"
+
+check:
+  type: CheckStream
+  stream_names:
+    - "workspace"
+
+spec:
+  type: Spec
+  documentationUrl: https://docs.captaindata.co/#1da39f5f-a26f-4a53-9e84-3e4b28077eae
```

Comment:
This is the Airbyte documentationUrl https://docs.airbyte.com/integrations/sources/captain-data